skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/BrevbakerService.kt

```diff
@@ -61,6 +66,25 @@ class BrevbakerServiceHttp(config: Config, authService: AuthService) : Brevbaker
         defaultRequest {
             url(brevbakerUrl)
         }
+        install(HttpRequestRetry) {
+            this.maxRetries = 3
+            delayMillis {
+                minOf(2.0.pow(it).toLong(), 1000L) + Random.nextLong(100)
+            }
+            retryOnExceptionIf { req, cause ->
+                if (req.method == HttpMethod.Post && req.url.build().segments.last() == "pdf") {
+                    return@retryOnExceptionIf false
+                }
+                val actualCause = cause.unwrapCancellationException()
+                val doRetry = actualCause is HttpRequestTimeoutException
+                        || actualCause is ConnectTimeoutException
+                        || actualCause is IOException
+                if (!doRetry) {
+                    logger.error("Won't retry for exception: ${actualCause.message}", actualCause)
```

Comment:
Hmm, vil vi nå potensielt få error logginnslag hvor vi ellers ikke ville fått det? F.eks om `actualCause is HttpClientException`.

Reply:
Litt usikker. Vi prøver, og så får vi leggje på meir filtrering viss det blir unødig støy.